tests/func/test_metrics.py

```diff
@@ -279,6 +279,33 @@ def test_formatted_output(self):
         assert expected_txt in stdout
         assert expected_json in stdout
 
+    def test_show_all_should_be_current_dir_agnostic(self):
+        os.chdir(self.DATA_DIR)
+
+        metrics = self.dvc.metrics.show(all_branches=True)
+        self.assertMetricsHaveRelativePaths(metrics)
+
+    def assertMetricsHaveRelativePaths(self, metrics):
+        root_relpath = os.path.relpath(self.dvc.root_dir, os.getcwd())
+        metric_path = os.path.join(root_relpath, "metric")
+        metric_json_path = os.path.join(root_relpath, "metric_json")
+        metric_json_tsv = os.path.join(root_relpath, "metric_tsv")
```

Comment:
Looks like `json` part was left from copypasting? :slightly_smiling_face:

Reply:
Maybe use `self.files` instead? Those are set in `setUp`.

Reply:
Guilty